core/src/main/java/io/aiven/klaw/helpers/db/rdbms/SelectDataJdbc.java

```diff
@@ -1489,43 +1489,50 @@ public boolean existsSchemaComponentsForEnv(String env, int tenantId) {
   public boolean existsComponentsCountForTeam(Integer teamId, int tenantId) {
     return Stream.<Supplier<Boolean>>of(
             () -> {
-              boolean res = schemaRequestRepo.existsRecordsCountForTeamId(teamId, tenantId);
+              boolean res =
+                  schemaRequestRepo.existsByTeamIdAndTenantIdAndRequestStatus(
+                      teamId, tenantId, RequestStatus.CREATED.value);
```

Comment:
adding request status parameter intentional ?

Reply:
Yes it was hard coded in the native query for the requests it is checking if there is an open request for that team